src/routes/solid-router/reference/response-helpers/redirect.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: redirect
+---
+
+Redirects to the next route.
+When done over a server RPC call, the redirect will be done through the server.
```

Comment:
could you specify what an RPC call is before abbreviating?

Reply:
thank you for this, I actually caught a redundancy with the acronym 

```suggestion
When done over a server RPC (Remote Procedure Call), the redirect will be done through the server.
```